discovery/wh_discovery.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 type webhookResponse struct {
-	Address string  `json:"address,omitempty"`
+	Address string  `json:"address"`
```

Comment:
I don't think this does much since `omitempty` only applies on encoding, not decoding, and the JSON parser will happy deserialize an object that has an empty `address` field